src/status_im/ui/screens/wallet/manage_connections/views.cljs

```diff
@@ -1,26 +1,42 @@
 (ns status-im.ui.screens.wallet.manage-connections.views
-  (:require [re-frame.core :as re-frame]
+  (:require [quo.core :as quo]
+            [quo.design-system.colors :as colors]
             [quo.react-native :as rn]
+            [re-frame.core :as re-frame]
+            [reagent.core :as reagent]
+            [status-im.ui.components.bottom-panel.views :as bottom-panel]
+            [status-im.ui.components.icons.icons :as icons]
             [status-im.ui.components.react :as react]
+            [status-im.ui.screens.wallet-connect.session-proposal.views :refer [app-management-sheet-view]]
             [status-im.ui.screens.wallet.manage-connections.styles :as styles]
-            [quo.core :as quo]
-            [quo.design-system.colors :as colors]
-            [status-im.ui.components.icons.icons :as icons]
             [status-im.utils.utils :as utils]))
 
-(defn print-session-info [session]
+(defn account-selector-bottom-sheet [session show-account-selector?]
+  (when @show-account-selector?
```

Comment:
its better to move condition outside the component, because it will be more readable, here you just hide condition and reaction